packages/flutter/lib/src/gestures/events.dart

```diff
@@ -2036,7 +2036,9 @@ mixin _CopyPointerCancelEvent on PointerEvent {
   }
 }
 
-/// The input from the pointer is no longer directed towards this receiver.
+/// The input from the pointer is no longer directed towards this receiver. The
```

Comment:
According to the guide, the doc should starts with a short sentence. 
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements
Try to rephrase what you want to add in a second paragraph. 

Also, the sentence 'The "receiver" in this context does not refer to a widget, but the entire client' sounds kind of awkward out of context. What is this intended to explain or avoid?